templates/cloudflare-workers/package.json

```diff
@@ -3,31 +3,30 @@
   "sideEffects": false,
   "scripts": {
     "build": "remix build",
-    "deploy": "wrangler publish",
-    "dev:remix": "remix watch",
-    "dev:miniflare": "cross-env NODE_ENV=development miniflare ./build/index.js --watch",
-    "dev": "remix build && run-p \"dev:*\"",
-    "start": "cross-env NODE_ENV=production miniflare ./build/index.js"
+    "dev": "npm run build && concurrently \"npm:dev:remix\" \"npm:dev:cloudflare\"",
```

Comment:
We're using `npm-run-all` in all other templates/examples, so maybe we should use that here as well to keep it consistent?